### PR TITLE
Closes #1793: Add Lifecycle.addObservers to ktx

### LINKED ISSUE #1793: Add Lifecycle.addObservers to support-ktx
This extension function is duplicated across the sample, reference and fenix browsers.

Extensify it!

### PR DESCRIPTION
**Pull request by jonalmeida**

### CODE REVIEW

File: components/support/ktx/src/main/java/mozilla/components/support/ktx/android/arch/lifecycle/Lifecycle.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+package mozilla.components.support.ktx.android.arch.lifecycle
+
+import android.arch.lifecycle.Lifecycle
+import android.arch.lifecycle.LifecycleObserver
+
+/**
+ * Calls [Lifecycle.addObserver] for a variable list of [LifecycleObserver]s.
+ */
+fun Lifecycle.addObservers(vararg observers: LifecycleObserver) = observers.forEach { addObserver(it) }
```
Comment (pocmo) [inline components/support/ktx/src/main/java/mozilla/components/support/ktx/android/arch/lifecycle/Lifecycle.kt]: Test? :)

File: docs/changelog.md [hunk 2]
```diff
@@ -35,13 +35,24 @@ permalink: /changelog/
...
+  * Added `Lifecycle.addObservers` to observe the lifecycle for multiple classes.
+  ```kotlin
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    lifecycle.addObservers(
+      fullscreenFeature,
+      sessionFeature,
+      customTabsToolbarFeature
+    )
+  }
+  ```
```
Comment (pocmo) [inline docs/changelog.md]: This is the wrong version - this should go in 0.40.0. :)

File: components/support/ktx/src/main/java/mozilla/components/support/ktx/android/arch/lifecycle/Lifecycle.kt (hunk 1 above)
Comment (jonalmeida) [reply components/support/ktx/src/main/java/mozilla/components/support/ktx/android/arch/lifecycle/Lifecycle.kt]: I wasn't sure what to test here.

`verify(lifecycle).addObserver()`?

File: docs/changelog.md (hunk 2 above)
Comment (jonalmeida) [reply docs/changelog.md]: Oops! Thanks for the catch!

**Pull request merged**